action.yml

```diff
@@ -64,6 +64,13 @@ inputs:
       The GitHub token used to make authenticated API requests.
     default: ${{ github.token }}
     required: false
+  single-subject-attestations:
+    description: >
+      If true, generate one attestation per subject, 
+      otherwise generate a single attestations with multiple subjects.
```

Comment:
Default behaviour is multi subject can probably kill this, replace comma with a period.

```suggestion
```

Reply:
Done.